client/my-sites/site-settings/section-security.jsx

```diff
@@ -58,8 +60,9 @@ module.exports = React.createClass( {
 			<div>
 				<JetpackProtect site={ site } />
 				{
-					config.isEnabled( 'settings/security/scan' ) ?
-						<JetpackScan site={ site } /> : ''
+					config.isEnabled( 'settings/security/scan' )
+						? <JetpackScan site={ site } />
```

Comment:
I think we should use the `{ config.isEnabled( 'settings/security/scan' ) && <JetpackScan site={ site } /> } ` way here.